activerecord/lib/active_record/connection_adapters/sqlite3/schema_dumper.rb

```diff
@@ -0,0 +1,13 @@
+module ActiveRecord
+  module ConnectionAdapters
+    module SQLite3
+      module ColumnDumper
+        private
+
+          def default_primary_key?(column)
+            schema_type(column) == :integer
+          end
```

Comment:
Oh, I see. Thanks. This means that all third party adapters should choice whether implement default bigint pk (and implement `Compatibility::V5_0`) or override `def default_primary_key?(column) schema_type(column) == :integer; end`, right?